.evergreen/run-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+DRIVERS_TOOLS=${DRIVERS_TOOLS:-}
+MONGODB_VERSION=${MONGODB_VERSION:-}
+
+# install MongoDB
+# Functions to fetch MongoDB binaries
+. ${DRIVERS_TOOLS}/.evergreen/download-mongodb.sh
+
+get_distro
+if [ -z "$MONGODB_DOWNLOAD_URL" ]; then
+    get_mongodb_download_url_for "$DISTRO" "$MONGODB_VERSION"
+fi
+# Even though we have the MONGODB_DOWNLOAD_URL, we still call this to get the proper EXTRACT variable
+get_mongodb_download_url_for "$DISTRO"
+download_and_extract "$MONGODB_DOWNLOAD_URL" "$EXTRACT"
+
+# run tests
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+[ -z "$MARCH" ] && MARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
+
+if [ "$AUTH" != "noauth" ]; then
```

Comment:
Can we take strings like this and stick them in variables? I'm worried about things like `noauth` or `nossl` being changed in the future.

Reply:
I think it's a bit premature, there's only a single reference to it. I'll remove the whole section of code actually, since we aren't presently running auth or ssl tests and we aren't likely to use this pattern anyway.